modules/imgproc/src/smooth.cpp

```diff
@@ -3198,12 +3198,10 @@ class BilateralFilter_8u_Invoker :
     {
         int i, j, cn = dest->channels(), k;
         Size size = dest->size();
-        #if CV_SSE3
+#if CV_SIMD128
         int CV_DECL_ALIGNED(16) buf[4];
-        float CV_DECL_ALIGNED(16) bufSum[4];
-        static const unsigned int CV_DECL_ALIGNED(16) bufSignMask[] = { 0x80000000, 0x80000000, 0x80000000, 0x80000000 };
-        bool haveSSE3 = checkHardwareSupport(CV_CPU_SSE3);
```

Comment:
We need to ask about SSE3 presence for the cases when a user has no SSE3 on their CPU.

Reply:
Not anymore.

OpenCV checks available features and [fails on problems](https://github.com/opencv/opencv/blob/3.3.0/modules/core/src/system.cpp#L506-L519).

Reply:
Thanks, other my comments about SSE3 are not actual anymore